src/api/todos.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { client } from '../utils/fetchClient';
+import { Todo } from '../types/Todo';
+
+export const getTodos = () => {
+  return client.get<Todo[]>('/todos?userId=10222');
+};
+
+export const postTodo = (data: any) => {
+  return client.post('/todos?userId=10222', data);
```

Comment:
The query parameter 'userId' should be added in the 'data' object instead of being appended to the URL.